vmware_rest_code_generator/templates/header.j2

```diff
@@ -73,10 +74,11 @@ def prepare_argument_spec():
             fallback=(env_fallback, ['VMWARE_REST_LOG_FILE']),
         ),
         "session_timeout": dict(
-            type='float',
+            type="float",
             default=300,
```

Comment:
```suggestion
```

Reply:
I think we should remove this default value so that we keep the current way of working

Reply:
ok.